SMCollection/schema-LD.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "schemaVersion": "0.0",
+  "$id": "https://raw.githubusercontent.com/smart-data-models/dataModel.SocialMedia/master/SMCollection/schema.json",
+  "title": "Social Media - SMCollection",
+  "description": "This entity contains a harmonised description of a generic SMCollection made for the Social Media domain. This entity is primarily associated with the process of collection of Social Media posts (primarily Twitter).",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
+    },
+    {
+          "$ref": "https://forge.etsi.org/rep/NGSI-LD/NGSI-LD/raw/master/schema/Entity.json#/definitions/Relationship"
+    },
+    {
+      "properties": {
+	      
+	  "type": {
+          "type": "string",
+          "description": "Property. NGSI-LD Entity Type. It must be equal to SMCollection.",
+          "enum": [
+            "SMCollection"
+          ]
+        },
+	      
+	  "description": {
+          "type": "string",
+          "description": "Property.  Model:'https://schema.org/Text'. Units:'No unit'. General description of the SMCollection."
+        },
+	      
+		        "hasPosts": {
+          		"type": "array",
+	                "items": {
+          		"$ref": "https://forge.etsi.org/rep/NGSI-LD/NGSI-LD/raw/master/schema/Entity.json#/definitions/Relationship"
+	  			}
+       			 },
+	      
+	                "hasAnalysis": {
+          		"type": "array",
+	  		"items": {
+          		"$ref": "https://forge.etsi.org/rep/NGSI-LD/NGSI-LD/raw/master/schema/Entity.json#/definitions/Relationship"
+	  			 }
+        			},
+	      
+	      "groupedAt": {
+	      "type": "string",
+	      "format" : "date-time",
+	      "description": "Property. The date and time of when the collection was constructed / grouped."
+      	}
+      }
+    }
+  ],
+  "required": [
+    "id",
+    "type",

```

Comment:
possibly excessive required properties

Reply:
This is what was proposed, I will look into it again.